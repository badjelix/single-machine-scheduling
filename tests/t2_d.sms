20
0 2 2 1 2
9 2 11 2 1 1
9 3 15 1 3
6 1 7 1 1
15 12 51 3 3 4 5
2 4 14 3 1 2 1
6 2 10 2 1 1
17 7 38 5 1 1 2 2 1
1 6 7 4 1 1 2 2
7 12 19 3 4 3 5
18 10 38 3 6 1 3
13 4 25 3 2 1 1
11 9 38 5 1 2 1 2 3
11 1 14 1 1
6 8 22 2 5 3
7 7 28 3 3 2 2
3 6 21 2 2 4
18 10 28 3 5 2 3
9 3 12 2 1 2
0 9 27 5 1 2 2 2 2
0
0
0
0
2 9 15
0
0
1 17
0
1 1
2 9 2
1 7
0
1 1
0
2 9 2
1 4
0
1 6
2 4 9
