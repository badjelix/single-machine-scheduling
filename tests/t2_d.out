10
1 0
2 9 10
4 6
6 2 3 5
7 7 8
8 18 23 24 26 28
12 19 21 22
13 29 30 32 33 35
14 11
17 12 14
